doc/release-notes/0.41/0.41.md

```diff
@@ -68,6 +68,36 @@ The following table covers notable changes in v0.41.0. Further information about
+</tr>
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/17643">#17643</a></td>
+<td valign="top">New <tt>-XX:ContinuationCache</tt> option is added to tune the continuation tier 1 and 2 cache size.</tt></td>
+<td valign="top">OpenJDK 21 and later</td>
+<td valign="top">New continuations can reuse the cached structure instead of allocating new memory for it. The continuation cache is implemented in two tiers, tier 1 and tier 2. You can set the maximum size for the two tiers with the <tt>-XX:ContinuationCache</tt> option. If an application uses more than 10000 virtual threads, setting a larger cache size might improve performance.</td>
+</tr>
+<tr>
+<td valign="top"><a href="https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/265">#265</a></td>
+<td valign="top">New <tt>-XX:[+|-]UseZlibNX</tt> option is added to control the loading of the <tt>zlibnx</tt> library in the <tt>LIBPATH</tt> environment variable.</td>
+<td valign="top">All versions (AIX&reg;)</td>
+<td valign="top">The AIX system adds the <tt>zlibNX</tt> library location, if available, to the <tt>LIBPATH</tt> variable by default. But, having the <tt>zlibNX</tt> library directory location in the <tt>LIBPATH</tt> variable might cause some issues. You can disable adding of the <tt>zlibNX</tt> library location to that variable with the <tt>-XX:-UseZlibNX</tt> option.</td>
+</tr>
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/13686">#13686</a></td>
+<td valign="top">OpenSSL 3.x is now supported on all operating systems.</td>
+<td valign="top">All versions</td>
+<td valign="top">In the earlier releases, OpenSSL 3.x was supported only on Linux. Now, OpenSSL 3.x is supported on all operating systems.</td>
+</tr>
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/18202">#18202</a></td>
+<td valign="top">Support is added for the <tt>com.sun.management.ThreadMXBean.getThreadAllocatedBytes()</tt> API</td>
+<td valign="top">All versions (All operating systems except z/OS&reg;)</td>
+<td valign="top">In the earlier releases, thread memory allocation measurement was not supported. With this release, the OpenJ9 VM implementation supports thread memory allocation measurement (<tt>com.sun.management.ThreadMXBean.getThreadAllocatedBytes()</tt> API). The <tt>isThreadAllocatedMemorySupported()</tt>, <tt>setThreadAllocatedMemoryEnabled()</tt>, and <tt>isThreadAllocatedMemoryEnabled()</tt> methods are supported now.</td>
```

Comment:
So can I say 
> The <tt>isThreadAllocatedMemorySupported()</tt> method now returns true instead of false. You can also enable the thread memory allocation measurement with the <tt>setThreadAllocatedMemoryEnabled()</tt> method and then the <tt>isThreadAllocatedMemoryEnabled()</tt> method also returns true. These methods do not throw the "UnsupportedOperationException" error now.

Reply:
You don't need to call setThreadAllocatedMemoryEnabled() since isThreadAllocatedMemoryEnabled() return true by default now. It could be like the following:

The isThreadAllocatedMemorySupported() method now returns true instead of false. The setThreadAllocatedMemoryEnabled() and isThreadAllocatedMemoryEnabled() methods do not throw the "UnsupportedOperationException" error now, and default to being enabled.